terraform/stacks/main/stack.tf

```diff
@@ -305,6 +305,15 @@ module "diego" {
   log_bucket_name = module.log_bucket.elb_bucket_name
 }
 
+module "opensearch_logs_customer" {
+  source = "../../modules/opensearch_domain"
+  instance_type= "t3.medium.search"
+  private_elb_subnets = [module.cf.services_subnet_az1, module.cf.services_subnet_az2]
+  domain= "{var.stack_description}-logs-customer"
+  master_user_name = "test_user"
+  master_password = "CHANGEME"
```

Comment:
Let's wire these up to come from the pipeline and make sure not to hardcode any values, even for testing